### PR TITLE
refactor: Use dotenv for configuration

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

**Comment by bennyn** [comment]
Do we still need the `baseData` fallback when having `.env` files?

**Comment by ffflorian** [comment]
> Do we still need the `baseData` fallback when having `.env` files?

Yes, because it's not a fallback but a Grunt configuration :slightly_smiling_face:

File: .copyconfigrc.js [hunk 1]
```diff
@@ -10,11 +10,11 @@ const repositoryUrl = pkg.dependencies[configurationEntry];
...
-    [path.join(logoSource, '32x32.png')]: 'resources/icons/32x32.png',
-    [path.join(logoSource, '256x256.png')]: ['resources/icons/256x256.png', 'electron/img/logo.256.png'],
-    [path.join(logoSource, 'logo.ico')]: 'electron/img/logo.ico',
-    [path.join(macOsSource, '**')]: 'resources/macos/',
-    [path.join(pkg.name, '.env.defaults')]: '.env.defaults',
+    [`${logoSource}/32x32.png`]: 'resources/icons/32x32.png',
+    [`${logoSource}/256x256.png`]: ['resources/icons/256x256.png', 'electron/img/logo.256.png'],
+    [`${logoSource}/logo.ico`]: 'electron/img/logo.ico',
+    [`${macOsSource}/**`]: 'resources/macos/',
+    [`${pkg.name}/.env.defaults`]: '.env.defaults',
```
Comment (lipis) [inline .copyconfigrc.js]: Does this thing work on Windows?

Comment (ffflorian) [reply .copyconfigrc.js]: Of course :slightly_smiling_face:

**Pull request merged**